shiny-ci-cd/actions/ci-cd.yml

```diff
@@ -6,46 +6,63 @@ name: CI-CD
 
 jobs:
   CI-CD:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.config.os }}
+
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+
+    strategy:
+      # we keep a matrix for convenience, but we would always want to run on one
```

Comment:
Why don't we have this comment and section also in the renv case?

Reply:
But we do? Either way, I will later align these sentences to what we changed in the ShinyCICD repo.